openslides_backend/action/actions/meeting/clone.py

```diff
@@ -291,23 +291,7 @@ def build_write_request_helper(
         )
 
     def check_permissions(self, instance: Dict[str, Any]) -> None:
-        meeting = self.datastore.fetch_model(
-            FullQualifiedId(Collection("meeting"), instance["meeting_id"]),
-            ["committee_id"],
-        )
-        committee_id = meeting["committee_id"]
-        if not has_committee_management_level(
-            self.datastore,
-            self.user_id,
-            CommitteeManagementLevel.CAN_MANAGE,
-            committee_id,
-        ):
-            raise PermissionDenied(
-                f"Missing {CommitteeManagementLevel.CAN_MANAGE.get_verbose_type()}: {CommitteeManagementLevel.CAN_MANAGE} for committee {committee_id}"
-            )
-        if (
-            payload_committee_id := instance.get("committee_id")
-        ) and payload_committee_id != committee_id:
+        if payload_committee_id := instance.get("committee_id"):
```

Comment:
`committee_id` is not required in the payload. As stated in the linked issue, if it is not given, the target meeting is equivalent to the source meeting and the permission check has to be done as before.

Reply:
Ah, I understand.

Reply:
If a committee_id is in the payload, I use this for the check, else I would use the committee_id from the meeting.